user/libgnomekbd/template.py

```diff
@@ -0,0 +1,26 @@
+pkgname = "libgnomekbd"
+pkgver = "3.28.1"
+pkgrel = 0
+build_style = "meson"
+configure_args = ["-Dintrospection=false"]
+hostmakedepends = [
+    "meson",
+    "gettext",
+    "pkgconf",
+    "glib-devel",
+    "gobject-introspection",
+]
+makedepends = ["gtk+3-devel", "libxklavier-devel"]
+pkgdesc = "Gnome keyboard configuration library"
+maintainer = "Earldridge Jazzed Pineda <earldridgejazzedpineda@gmail.com>"
+license = "GPL-2.0-or-later"
+url = "https://gitlab.gnome.org/GNOME/libgnomekbd"
+source = f"$(GNOME_SITE)/{pkgname}/{pkgver[:-2]}/{pkgname}-{pkgver}.tar.xz"
```

Comment:
```suggestion
source = f"$(GNOME_SITE)/libgnomekbd/{pkgver[:-2]}/libgnomekbd-{pkgver}.tar.xz"
```

Reply:
For your information, I used the template from an attempt to package the Budgie desktop environment (#172), and modified it to make it compilable once again.

Reply:
sure, that's okay :)